File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_api/dto/inventory/rice_variety/RiceVarietyResponseDTO.java
package com.example.fieldsync_inventory_api.dto.inventory.rice_variety;

import lombok.*;

import java.time.Instant;

// For sending data to client (read)
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RiceVarietyResponseDTO {
    private Integer id;
    private String name;
    private String description;
    private String imageUrl;
    private Instant createdAt;
    private Instant updatedAt;
    private Instant deletedAt;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/rice_variety/RiceVarietyResponseDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.rice_variety

// RiceVarietyResponseDto for the server's response
data class RiceVarietyResponseDto(
    val id: Int,
    val name: String,
    val description: String,
    val imageUrl: String?,
    val createdAt: String,
    val updatedAt: String,
    val deletedAt: String?
)
